NickvisionMoney.Shared/Resources/Strings.zh.resx

```diff
@@ -0,0 +1,643 @@
+	</resheader>
+
+	<resheader name="version">
+		<value>2.0</value>
+	</resheader>
+
+	<resheader name="reader">
+		<value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+	</resheader>
+
+	<resheader name="writer">
+		<value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+	</resheader>
+
+	<!--Use: &#xA; for newline-->
+
+	<data name="Description" xml:space="preserve">
+		<value>個人財政記錄軟體</value>
+	</data>
+
+	<data name="Version" xml:space="preserve">
+		<value>版本: {0}</value>
+	</data>
+
+	<data name="About" xml:space="preserve">
+		<value>關於{0}</value>
+	</data>
+
+	<data name="Changelog" xml:space="preserve">
+		<value>變更日誌</value>
+	</data>
+
+	<!--Credits-->
+	<data name="Credits" xml:space="preserve">
+		<value>鳴謝</value>
+	</data>
+
+	<data name="Developers.Credits" xml:space="preserve">
+		<!--DEVELOPER NAME(S) GOES HERE. SEPARATE WITH &#xA;-->
+		<value>Nicholas Logozzo https://github.com/nlogozzo&#xA;GitHub上的開發者 ❤️ https://github.com/nlogozzo/NickvisionMoney/graphs/contributors</value>
+	</data>
+
+	<data name="Designers.Credits" xml:space="preserve">
+		<!--DESIGNER NAME(S) GOES HERE. SEPARATE WITH &#xA;-->
+		<value>Nicholas Logozzo https://github.com/nlogozzo&#xA;Fyodor Sobolev https://github.com/fsobolev</value>
+	</data>
+
+	<data name="Artists.Credits" xml:space="preserve">
+		<!--ARTIST NAME(S) GOES HERE. SEPARATE WITH &#xA;-->
+		<value>David Lapshin https://github.com/daudix-UFO</value>
+	</data>
+
+	<data name="Translators.Credits" xml:space="preserve">
+		<!--TRANSLATOR NAME(S) GOES HERE. SEPARATE WITH &#xA;-->
+		<value></value>
+	</data>
+
+	<!--Actions-->
+	<data name="OK" xml:space="preserve">
+		<value>OK</value>
+	</data>
+
+	<data name="Yes" xml:space="preserve">
+		<value>是</value>
+	</data>
+
+	<data name="No" xml:space="preserve">
+		<value>否</value>
+	</data>
+
+	<data name="Cancel" xml:space="preserve">
+		<value>取消</value>
+	</data>
+
+	<data name="New" xml:space="preserve">
+		<value>新增</value>
+	</data>
+
+	<data name="Open" xml:space="preserve">
+		<value>開啟</value>
+	</data>
+
+	<data name="Close" xml:space="preserve">
+		<value>關閉</value>
+	</data>
+
+	<data name="Save" xml:space="preserve">
+		<value>儲存</value>
+	</data>
+
+	<!--Pages-->
+	<data name="Home" xml:space="preserve">
+		<value>目錄</value>
+	</data>
+
+	<data name="Account" xml:space="preserve">
+		<value>帳戶</value>
+	</data>
+
+	<data name="Settings" xml:space="preserve">
+		<!--Mostly used in WinUI-->
+		<value>設定</value>
+	</data>
+
+	<data name="Preferences" xml:space="preserve">
+		<!--Mostly used in GNOME-->
+		<value>偏好設定</value>
+	</data>
+
+	<data name="KeyboardShortcuts" xml:space="preserve">
+		<value>快捷鍵</value>
+	</data>
+
+	<!--MainWindow-->
+	<data name="NickvisionMoneyAccount" xml:space="preserve">
+		<value>Nickvision的帳戶</value>
```

Comment:
Here "Money" is part of the application name "Nickvision Money", so IMO "Money" should be kept untranslated

Reply:
"帳戶" means "bank account". If it translates to "Nickvision's money bank account" it'd be weird. Would "Nickvision's application 'Money' Account" be better?

Reply:
> If it translates to "Nickvision's money bank account" it'd be weird.

I don't see how it's weird. Because it's not just "money" here, it's "Money" - the name of the app.
We're going to rename the app to Denaro soon. So it will be "Denaro Account" there. Would it be weird to translate as "Denaro bank account"? By the same logic, yes, because "denaro" means "money" in Italian. But it doesn't matter - because it's a name, and its meaning in original language isn't valid in the context. Same goes to "Money" currently.
However, I'm not a professional translator, that's my opinion as hobbyist (before software I had been translating comic books 😊 )

Reply:
@nlogozzo I request your opinion, please

Reply:
> I don't see how it's weird. Because it's not just "money" here, it's "Money" - the name of the app.

I get you but using mixed Chinese and English is often complained by users. Denaro is more clearly a name, while Money is a term. What I suggested there - specifying it as a name instead of a term, prevents that situation.

Reply:
Has been updated to include the name "Money" but with "software" prefix to prevent confusion.